manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -223,6 +224,21 @@ type ChartSpec struct {
 	Values map[string]string `json:"values,omitempty"`
 }
 
+// FybrikModuleStatus defines the observed state of FybrikModule.
+type FybrikModuleStatus struct {
+	// ErrorMessage indicates that an error has happened during the reconcile, unrelated to a specific asset
```

Comment:
Wrong comment, it's an error message received during the module validation.

Reply:
Fixed